src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/OutOfTimeDecisionDetailsAppender.java

```diff
@@ -2,20 +2,24 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import lombok.Getter;
-import lombok.Setter;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.OutOfTimeDecisionDetails;
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.IdValue;
 
 @Service
-@Setter
-@Getter
 public class OutOfTimeDecisionDetailsAppender {
 
     private List<IdValue<OutOfTimeDecisionDetails>> allOutOfTimeDecisionDetails;
 
-    public List<IdValue<OutOfTimeDecisionDetails>> append(
+    public synchronized List<IdValue<OutOfTimeDecisionDetails>> getAllOutOfTimeDecisionDetails() {
```

Comment:
If we use allOutOfTimeDecisionDetails as a local variable in the append method. Do we need to use the getter and setter? im not sure if we need to hold the temporary state, but if we do we can use a request scope instead of synchronized?